exp/api/v1beta1/types.go

```diff
@@ -144,3 +144,21 @@ func ConvertToSdkLinuxNodeConfig(linuxNodeConfig *LinuxNodeConfig) *containerpb.
 	}
 	return &sdkLinuxNodeConfig
 }
+
+// ConvertToSdkNetworkPolicy converts NetworkPolicy config to a value that is used by GCP SDK.
+func ConvertToSdkNetworkPolicy(networkPolicy *NetworkPolicy) *containerpb.NetworkPolicy {
```

Comment:
Following on from a previous comment can this be moved to a different package.

Reply:
Done